ui/react-components/components/Location/LocationSearch.jsx

```diff
@@ -2,8 +2,8 @@ import Dropdown from "../Dropdown/Dropdown.jsx";
 import React, {useEffect, useState} from "react";
 import PropTypes from "prop-types";
 import {injectIntl} from 'react-intl';
-// import {getAllByTag} from "../../api/locationApi";
-import {getAllByTag} from "../../api/__mocks__/locationApi";
+import {getAllByTag} from "../../api/locationApi";
+// import {getAllByTag} from "../../api/__mocks__/locationApi";
```

Comment:
We should remove the commented code. Unless, the plan is to toggle between mock and actual at times.